settings/environ/base.py

```diff
@@ -27,7 +28,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [fam-kyagulanyi-cms/Templates],
```

Comment:
maintain small letters while naming files.. 
In Python capital letters are limited to `class fn()` in small letters like
`templates` instead of `Templates`

Reply:
ok

Reply:
Resolve the merge conflicts by merging your branch into master